pkg/gofr/service/new.go

```diff
@@ -137,12 +137,14 @@ func (h *httpService) createAndSendRequest(ctx context.Context, method string, p
 	uri := h.url + "/" + path
 	uri = strings.TrimRight(uri, "/")
 
-	spanContext, span := h.Tracer.Start(ctx, uri)
+	ctx, span := h.Tracer.Start(ctx, uri)
 	defer span.End()
 
-	spanContext = httptrace.WithClientTrace(spanContext, otelhttptrace.NewClientTrace(ctx))
+	// Attach client-side trace handling for HTTP request
+	clientTraceCtx := httptrace.WithClientTrace(ctx, otelhttptrace.NewClientTrace(ctx))
 
-	req, err := http.NewRequestWithContext(spanContext, method, uri, bytes.NewBuffer(body))
+	// Create the HTTP request with the tracing context
```

Comment:
Here and everywhere we have added new comments, we should make sure that they end with a period (.)